packages/storage-mixin/lib/backends/secure.js

```diff
@@ -210,7 +208,7 @@ SecureBackend.prototype.findOne = function(model, options, done) {
  */
 SecureBackend.prototype.find = function(collection, options, done) {
   debug('Fetching data...', collection.length);
-  keytar
+  require('keytar')
```

Comment:
> this is making much of a difference anymore

Ah, what changed? This should avoid the necessity of rebuilds unless this code actually runs

Reply:
I’m just thinking about the fact that we’re fairly close to bumping Electron already, which iiuc would mean that we don’t need to rebuild here because we’re using the Node-API variant of keytar?

Reply:
As far as i remember we can't move on with the keytar update that uses NAPI cause of centos 7 or something, @Anemy is that true?

Reply:
@mcasimir We can still move ahead with this as long as we recompile it ourselves, once, in CI.

Reply:
Ah ok there is a better way around! Let's get rid of this change then. Is this trick already in some branch / do we have a ticket for that?

Reply:
No, I don’t think so – let me poke around a bit to see if it’s easy enough to do

Reply:
Yeah, looks like it’s going to depend on the Electron bump work. I’ve add a ticket to that project: https://jira.mongodb.org/browse/COMPASS-5138